provisioner/components/imagerepo/setup_image_depot.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing, please email
+# opensource@seagate.com or cortx-questions@seagate.com.
+
+import os
+from provisioner.const import PATH
+
+
+def createDir():
+    try:
+        if os.system("ls %s" %(PATH.IMAGE_TAR_FILE_PATH)) != 0:
+            os.system("mkdir %s" %(PATH.IMAGE_TAR_FILE_PATH))
+            print("Directory Created %s" %PATH.IMAGE_TAR_FILE_PATH) #TODO: Replace print with log
+    except Exception as e:
+        print(f'{e}') #TODO: Replace print with log
+
+def loadImages():
+    try:
+        img = os.popen("ls %s*.tar" %(PATH.IMAGE_TAR_FILE_PATH)).read()
```

Comment:
Please check variable name "img",
It seems like from single image you're getting list of images.
Can be renamed as
"img"  -> image_file
"imgList" -> img_list
"i" -> img

Reply:
Changed variable names

Reply:
Thanks!